fastlane/Fastfile

```diff
@@ -3,9 +3,10 @@
 default_platform(:ios)
 
 platform :ios do
-  desc 'Builds the project and runs tests'
+  desc 'Builds the Swift package and runs tests'
   lane :test do
```

Comment:
This fails for me locally with 

```
[06:08:50]: ▸ xcodebuild: error: The workspace named "Gridicons" does not contain a scheme named "Gridicons". The "-list" option can be used to find the names of the schemes in the workspace.
```

However, it works in CI 🤔 I wonder what the difference is between the two setups. The only thing I can think of is Intel vs Apple Silicon, but that doesn't sound related to a scheme not being found.